addon/helpers/bs-form-horiz-input-class.js

```diff
@@ -6,8 +6,14 @@ export default helper(function bsFormHorizInputClass([horizontalLabelGridClass]
   if (isBlank(horizontalLabelGridClass)) {
     return undefined;
   }
-  let parts = horizontalLabelGridClass.split('-');
-  assert('horizontalInputGridClass must match format bootstrap grid column class', parts.length === 3);
-  parts[2] = 12 - parts[2];
-  return parts.join('-');
+
+  let originalClasses = horizontalLabelGridClass.split(' ');
+  let inputClasses = originalClasses.map((originalClass) => {
+    let parts = originalClass.split('-');
+    assert('horizontalInputGridClass must match format bootstrap grid column class', parts.length === 3);
```

Comment:
Disclaimer: Not related to this PR.

The assumption that all Bootstrap grid column classes having 3 parts is wrong. Bootstrap also supports columns without breakpoint: `col-3`. That is used for all breakpoints unless overruled by a breakpoint specific column class (e.g. `col-md-3`). Will verify if it actually breaks in that case and open a separate ticket to track.